admin/client/src/components/Toolbar/Toolbar.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
This makes the component stateful, and less reuseable - it should be implemented on the container instead (`<CampaignAdmin>`) - having a look at refactoring that now